src/components/constants/url.js

```diff
@@ -1,4 +1,5 @@
 const BASE_URL = `${process.env.NEXT_PUBLIC_BASE_URL}`;
 export const SUBSCRIBE_TO_CHALLENGE_URL = `${BASE_URL}/challenges/subscribe`;
 export const CHALLENGES_URL = `${BASE_URL}/challenges`;
-export const USER_SELF = `${BASE_URL}/users/self`;
\ No newline at end of file
+export const USER_SELF = `${BASE_URL}/users/self`;
+export const ALL_USERS = `${BASE_URL}/members`;
```

Comment:
All members are users, but all users are not members. Please update this constant.

Reply:
cc @Maheima @rupal-2505